assets/src/application/ui/forms/espressoForm/renderers/FieldRenderer.tsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import classnames from 'classnames';
```

Comment:
Seems that it was used as `classNames`
![image](https://user-images.githubusercontent.com/1477453/74943882-ca0e7d80-53fd-11ea-804e-eaffdbe9eee0.png)

Reply:
Updated